include/config.py

```diff
@@ -102,7 +102,7 @@
 # (b bit of address = logN, N is the number of nodes)
 
 # Change here - Specify the Node parameters here
-num_tile_compute =7 # number of tiles mapped by dnn (leaving input and output tiles)
+num_tile_compute = 2 # number of tiles mapped by dnn (leaving input and output tiles)
```

Comment:
```suggestion
num_tile_compute = 7 # number of tiles mapped by dnn (leaving input and output tiles)
```

Just to reduce changes.